colors/edge.vim

```diff
@@ -150,10 +150,10 @@ if has('nvim')
   highlight! link LspDiagnosticsFloatingWarning WarningFloat
   highlight! link LspDiagnosticsFloatingInformation InfoFloat
   highlight! link LspDiagnosticsFloatingHint HintFloat
-  highlight! link LspDiagnosticsDefaultError ErrorText
-  highlight! link LspDiagnosticsDefaultWarning WarningText
-  highlight! link LspDiagnosticsDefaultInformation InfoText
-  highlight! link LspDiagnosticsDefaultHint HintText
+  highlight! link LspDiagnosticsDefaultError RedSign
+  highlight! link LspDiagnosticsDefaultWarning YellowSign
+  highlight! link LspDiagnosticsDefaultInformation BlueSign
+  highlight! link LspDiagnosticsDefaultHint GreenSign
```

Comment:
These highlight groups will change many other behaviors, you should use nvim-tree specific hi groups.

Should be fixed via https://github.com/sainnhe/edge/commit/8540e798b01fd6a3c80ffa4a1930d9733e294b34